lib/manager/gitlabci/extract.ts

```diff
@@ -120,15 +124,15 @@ export async function extractAllPackageFiles(
     if (is.array(doc?.include)) {
       for (const includeObj of doc.include) {
         if (is.string(includeObj.local)) {
-          const fileObj = includeObj.local.replace(/^\//, ''); // TODO #12071 #12070
+          const fileObj = includeObj.local.replace(regEx(/^\//), '');
           if (!seen.has(fileObj)) {
             seen.add(fileObj);
             filesToExamine.push(fileObj);
           }
         }
       }
     } else if (is.string(doc?.include)) {
-      const fileObj = doc.include.replace(/^\//, ''); // TODO #12071  #12070
+      const fileObj = doc.include.replace(/^\//, ''); // TODO  #12070
```

Comment:
the same regex is modified above
```suggestion
      const fileObj = doc.include.replace(regEx(/^\//), '');
```